### PR TITLE
SaaS Security text changes.

### PR DESCRIPTION
**Pull request by zhiwangprisma**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by eyalpalo** [comment]
Hi @zhiwangprisma, please see [this PR](https://github.com/demisto/content/pull/17036). There are changes made at the same places. Are you aware to this PR?

**Comment by saassecuritywriter** [comment]
> Hi @zhiwangprisma, please see [this PR](https://github.com/demisto/content/pull/17036). There are changes made at the same places. Are you aware to this PR?

Yes. Zhiwang and I are working together I this merge.

**Comment by eyalpalo** [comment]
@saassecuritywriter does it means that you guys will close one of the PRs?

**Comment by bakatzir** [comment]
@ilaner putting you on all of SaaS PRs.

**Comment by zhiwangprisma** [comment]
> 



> > Hi @zhiwangprisma, please see [this PR](https://github.com/demisto/content/pull/17036). There are changes made at the same places. Are you aware to this PR?
> 
> Yes. Zhiwang and I are working together I this merge.

yes, pls keep this pr. The other one should be closed

File: Packs/PrismaSaasSecurity/pack_metadata.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
     "name": "SaaS Security (Beta)",
+<<<<<<< HEAD
```
Comment (ShahafBenYakir) [inline Packs/PrismaSaasSecurity/pack_metadata.json:3]: remove
```suggestion
```

File: Packs/PrismaSaasSecurity/pack_metadata.json [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+    "description": "SaaS Security API is a cloud-based service that you can connect directly to your sanctioned SaaS applications using the cloud app’s API to provide data classification, sharing and permission visibility, and threat detection. This Content Pack provides insights into risks posed by data exposure and policy violations and enables you to use Cortex XSOAR to effectively manage the incidents discovered by SaaS Security API.",
+=======
```
Comment (ShahafBenYakir) [inline Packs/PrismaSaasSecurity/pack_metadata.json:5]: remove
```suggestion
```

File: Packs/PrismaSaasSecurity/pack_metadata.json [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
     "description": "Provides unparalleled visibility and precise control of SaaS applications using an extensive library of application signatures.",
+>>>>>>> master
```
Comment (ShahafBenYakir) [inline Packs/PrismaSaasSecurity/pack_metadata.json:7]: remove
```suggestion
```

**Comment by edik24** [comment]
Hi @zhiwangprisma 
you have various validation errors due to the changes you performed:

as Shahaf mentioned fix the metadata file:
remove lines 3, 5, 6.
Description appears twice - remove the wrong one
make the lines less than 130 chars (split the lines)

in SaasSecurity.yml
IncidentType and isFetch don't support additionalinfo field - it should be removed

if the integration set to "beta: true" then the display should be changed accordingly
 display: SaaS Security (Beta)
 
in incident Type json - the IncidentTypes/incidenttype-Saas_Security.json 
you should remove description - it's not supported there.

it is recommended to ran validation and lint locally before pushing the fixes, to verify that everything was fixed.

demisto-sdk validate -i *path to the pack on your hard drive*/Packs/PrismaSaasSecurity
and 
demisto-sdk lint -i *path to the pack on your hard drive*/Packs/PrismaSaasSecurity

File: Packs/PrismaSaasSecurity/pack_metadata.json (hunk 3 above)
Comment (zhiwangprisma) [reply Packs/PrismaSaasSecurity/pack_metadata.json]: updated

File: Packs/PrismaSaasSecurity/pack_metadata.json (hunk 2 above)
Comment (zhiwangprisma) [reply Packs/PrismaSaasSecurity/pack_metadata.json]: updated

**Review by ilaner — APPROVED** [review]
Great job

**Pull request merged**